src/main/java/uk/gov/hmcts/cmc/claimstore/services/search/CCDCaseDBI.java

```diff
@@ -14,28 +14,28 @@
 
 import static java.lang.String.format;
 
-@Service("caseRepository")
+@Service("caseDBI")
 @ConditionalOnProperty(prefix = "core_case_data", name = "api.url")
-public class CCDCaseRepository implements CaseRepository {
-    private final Logger logger = LoggerFactory.getLogger(CCDCaseRepository.class);
+public class CCDCaseDBI implements CaseDBI {
```

Comment:
Repository is the spring boot name for the data access class

I would prefer:
keep ClaimRepository and revert this back to what it currently was

Reply:
we will be discarding all claim names once we move to CCD, Claim->Case, ClaimService ->CaseService, and so fourth.

Reply:
At that point ClaimRepository will be deleted